### ISSUE TITLE
Wrong input instructions, raw output

### ISSUE THREAD

**Issue by ghostlyghastly** [Ekey/KG.Data.Tool#1]
Thank you!

The instructions seem wrong. Giving game path as input throws access error.

> Unhandled Exception: System.UnauthorizedAccessException: Access to the path 'D:\X\game' is denied.

Instead, the kfc_dir file path must be provided:

    ./KG.Unpacker.exe "D:/X/game/enshrouded.kfc_dir" "D:/EXTRACT/out"

The output is 102689 files + 11188 files in the '__Unknown' folder + resourceids.txt (every ID = '0') + version.txt

The files seem not converted and have no file ending, just sliced. This is useful but it'd be great if text could be converted to unicode, textures to png, audio to wave or ogg or whatever...

Text for example doesn't have newlines or string separators and I don't know how to handle this. See for example the credits in

46e14df6-fb8f-4f66-9463-e23b9e19e48e_fef918b8_0 / [46e14df6-fb8f-4f66-9463-e23b9e19e48e_fef918b8_0.zip](https://github.com/Ekey/KG.Data.Tool/files/14156719/46e14df6-fb8f-4f66-9463-e23b9e19e48e_fef918b8_0.zip)

**Comment by Ekey** [comment]
Yeah, I just forgot to add the DIR filename to Usage section, fixed anyway, thanks! :)

> The output is 102689 files + 11188 files in the '__Unknown' folder + resourceids.txt (every ID = '0') + version.txt
> 
> The files seem not converted and have no file ending, just sliced. This is useful but it'd be great if text could be converted to unicode, textures to png, audio to wave or ogg or whatever...

Archive doesn't contain filenames only their hashes. The list file contains ~95% of the real names (as far as I remember, there are about 10k unknowns names is left, so they are placed in the Unknown folder and named as hash).
PS: And yes, their real names look like GUIDs.
